parley/src/context.rs

```diff
@@ -20,7 +20,7 @@ use swash::text::cluster::CharInfo;
 use crate::builder::TreeBuilder;
 use crate::inline_box::InlineBox;
 
-/// Context for building a text layout.
+/// Shared scratch space used when constructing text layouts
```

Comment:
```suggestion
/// Shared scratch space used when constructing text layouts.
///
/// This type is designed to be a global resource with only one per-application (or per-thread).
```
See https://github.com/linebender/xilem/pull/405